src/app/[locale]/incident/summary/components/IncidentSummaryForm.tsx

```diff
@@ -194,3 +222,50 @@ const IncidentSummaryFormAttachments = ({
 }
 
 export { IncidentSummaryForm }
+
+const SubmitAlert = ({
```

Comment:
Is het een idee om deze componenten te verplaatsen naar losse bestanden? Anders wordt het component gigantisch en dat vind ik zelf vrij onoverzichtelijk.

Dit zijn specifieke componenten voor de summary overview dus ik zou deze ook gewoon in het mapje ```src/app/[locale]/incident/summary/components/``` stoppen.